packages/utilities/src/rtl.ts

```diff
@@ -20,9 +20,8 @@ export function getRTL(): boolean {
       setRTL(_isRTL);
     }
 
-    let doc = getDocument();
-    if (_isRTL === undefined && doc) {
-      _isRTL = doc.documentElement.getAttribute('dir') === 'rtl';
+    if (_isRTL === undefined) {
+      _isRTL = getDocumentRTL();
```

Comment:
I think it is unneeded to add another exported method to get RTL state.

Instead just get it from `body`, or if not present, from `documentElement`. The `body` element would have a higher precedence, so it should be evaluated first.

Reply:
I agree, I'll make the changes. Things evolved incorrectly from jest DOM confusion.